.env.example

```diff
@@ -37,3 +37,6 @@ ISOLATED_ST_UNSERIALIZATION_TIMEOUT_MILLIS=300
 
 # logging
 LOGGING_LEVEL=info
+
+# debug mode
+DEBUG=false
```

Comment:
I thought, shouldn't we use the existing `NODE_ENV=development` instead of a new one?